### PR TITLE
[enhance] add some example data for layers

### PR DESCRIPTION
**Pull request by jingpengw**

### CODE REVIEW

File: python/example.py [hunk 1]
```diff
@@ -47,18 +47,27 @@
...
 #Fith layer (vector graphics)
 viewer.add(data=[[10, 20 ,30],
                  [15, 20 ,30],
                  [15, 25 ,30]], name='layer_vector_graphics', layer_type='line')
 
+# Sixth layer (skeleton)
+skeleton = Skeleton([[10, 20, 30],
+                     [15, 20, 30],
+                     [15, 25, 30]], [[1,2],[2,3]])
+#viewer.add(name='skeleton', layer_type='single_mesh', data = skeleton)
```
Comment (william-silversmith) [inline python/example.py]: Why comment it out?

Comment (jingpengw) [reply python/example.py]: this test do not work, I don't know the correct format.

Comment (jingpengw) [reply python/example.py]: just removed the skeleton layer since the example do not work.

**Comment by jingpengw** [comment]
@tartavull can we merge this?

**Pull request merged**